docs/_snippets/shadow-dom-testing-library-in-preview.md

```diff
@@ -0,0 +1,34 @@
+```ts filename=".storybook/preview.ts" renderer="web-components" language="ts"
+import type { Preview } from '@storybook/web-components';
```

Comment:
When you can, you can adjust this import to align with the framework usage, which is now the default.
```suggestion
import type { Preview } from '@storybook/web-components-vite';
```